modules/claims_api/lib/bd/bd.rb

```diff
@@ -22,7 +22,15 @@ def search(claim_id, file_number)
       body = { data: { claimId: claim_id, fileNumber: file_number } }
       ClaimsApi::Logger.log('benefits_documents',
                             detail: "calling benefits documents search for claimId #{claim_id}")
-      client.post('documents/search', body)&.body&.deep_symbolize_keys
+      res = client.post('documents/search', body)&.body
+
+      raise ::Common::Exceptions::GatewayTimeout.new(detail: "Gateway timeout. #{res}") unless res.is_a?(Hash)
```

Comment:
not 100% sure, but we may not want the `res` in the response to the end client, we can/should possibly log it, but I presume not return to user? (logging is a gray area as some upstreams return PII so they shouldn't be logged, but I don't think BD is one of them)

Reply:
Gotcha. We can definitely take `res` out of the exception detail.

I thought `res` might be helpful in the logs (e.g., in the bug that prompted this change, we got `failed to request: timeout`, which hinted at the underlying problem). But since we possibly don't know every response the service could return, maybe it's prudent to not log the response?